### PR TITLE
RF | CreateNewViewWithExistingName

### PR DESCRIPTION
**Pull request by olpolezhaeva**
Added Before and AfterMethod.
Refactored locators.

### CODE REVIEW

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/CreateNewViewWithExistingNameTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import runner.BaseTest;
+
+public class CreateNewViewWithExistingNameTest extends BaseTest {
+
+    String nameOfView = "My new view";
```
Comment (zenicko) [inline src/test/java/CreateNewViewWithExistingNameTest.java:8]: See #721

File: src/test/java/CreateNewViewWithExistingNameTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public void createNewView() {
+        getDriver().findElement(By.xpath("//a[@title='New View']")).click();
+        getDriver().findElement(By.id("name")).sendKeys(nameOfView);
+        getDriver().findElement(By.xpath("//label[text()='My View']")).click();
+        getDriver().findElement(By.id("ok")).click();
+    }
+
+    @Test
+    public void testCreateNewViewWithAnExistingName() {
+
```
Comment (zenicko) [inline src/test/java/CreateNewViewWithExistingNameTest.java:19]: The empty line

**Review by zenicko — COMMENTED** [review]
Errors should never pass silently.

**Review by zenicko — APPROVED** [review]
Errors should never pass silently.

**Pull request merged**